blocks/components/editable/tinymce.js

```diff
@@ -3,7 +3,11 @@ export default class TinyMCE extends wp.element.Component {
 		this.initialize();
 	}
 
-	shouldComponentUpdate() {
+	shouldComponentUpdate( nextProps ) {
+		if ( this.editorNode.getAttribute( 'data-is-empty' ) !== nextProps.isEmpty ) {
```

Comment:
Alternatively, does the placeholder need to be shown on the TinyMCE element? Would save some trouble if we could render this in the parent `Editable`, perhaps on its wrapper, which I believe should have the same bounds?